internal/vds/vds.cpp

```diff
@@ -621,9 +545,16 @@ struct response metadata(
     meta["boundingBox"]["cdp"]  = bbox.world();
     meta["boundingBox"]["ilxl"] = bbox.annotation();
 
-    for (int i = 2; i >= 0 ; i--) {
-        meta["axis"].push_back(json_axis(i, layout));
-    }
+
+    const Axis inline_axis(axis_name::INLINE, layout);
```

Comment:
Same question with regards to do we always return this order or do we want to return the order from the VolumeDataLayout.

Reply:
Agree. That is a question that we should be respecting. I am not sure what is better from a user (of vds-slice) perspective. Is it nicer to guarantee a certain order information is handed to the user? On the other hand, are there users with different use cases that expect a different order than "Inline", "Crossline", "Sample"/"Depth"/"Time"?